DEVELOPMENT.md

```diff
@@ -125,8 +125,8 @@ A couple of key items in the mean time:
 Search the code for org.slf4j.LoggerFactory to see a sample of its use.
 
 ### Use of Java 8 features
-Quarks primary development environment is Java 8, to take advantage of lambda expressions
-since Quarks' primary api is a functional one.
+Edgent primary development environment is Java 8, to take advantage of lambda expressions
+since Edgent' primary api is a functional one.
```

Comment:
good eyes! thx.  I checked and this was the only instance of that transformation issue.